app/subscriber/src/features/search-page/components/advanced-search/components/sections/DateSection.tsx

```diff
@@ -36,6 +36,7 @@ export const DateSection: React.FC = () => {
             storeFilter({
               ...filter,
               startDate: date ? moment(date)?.startOf('day').toISOString() : '',
+              dateOffset: undefined,
```

Comment:
Required to clear out the replaced date offset, otherwise it isn't correctly saved.